components/hierarchical-view/hierarchical-view-styles.js

```diff
@@ -0,0 +1,49 @@
+import '../colors/colors.js';
+import { css } from 'lit-element/lit-element.js';
+
+export const hierarchicalViewStyles = css`
```

Comment:
The mixin and the styles will always be used together... what are your thoughts about moving the styles into the mixin's file to avoid an additional HTTP request?

Reply:
Agreed. I'm hoping very soon we'll be able to do a build that doesn't do bundling, so everything would be loaded at runtime. That means that imports that don't need to be imports (they're used 1:1 with another file and never referenced elsewhere) should be merged.

Reply:
That makes sense to me